diff --git a/node_modules/@astrojs/starlight/utils/head.ts b/node_modules/@astrojs/starlight/utils/head.ts
index 58def2e..70cedd4 100644
--- a/node_modules/@astrojs/starlight/utils/head.ts
+++ b/node_modules/@astrojs/starlight/utils/head.ts
@@ -16,7 +16,6 @@ export function getHead(
 	siteTitle: string
 ): HeadConfig {
 	const { data } = entry;
-
 	const canonical = context.site ? new URL(context.url.pathname, context.site) : undefined;
 	const canonicalHref = canonical?.href
 		? formatCanonical(canonical.href, {
@@ -106,7 +105,6 @@ export function getHead(
 			},
 		});
 	}
-
 	return createHead(headDefaults, config.head, data.head);
 }
 
@@ -114,6 +112,7 @@ export function getHead(
 function createHead(defaults: HeadUserConfig, ...heads: HeadConfig[]) {
 	let head = HeadSchema.parse(defaults);
 	for (const next of heads) {
+		if(!next) continue; 
 		head = mergeHead(head, next);
 	}
 	return sortHead(head);
diff --git a/node_modules/@astrojs/starlight/utils/navigation.ts b/node_modules/@astrojs/starlight/utils/navigation.ts
index 5fe09b8..6465e2a 100644
--- a/node_modules/@astrojs/starlight/utils/navigation.ts
+++ b/node_modules/@astrojs/starlight/utils/navigation.ts
@@ -240,7 +240,7 @@ function treeify(routes: Route[], locale: string | undefined, baseDir: string):
 	const treeRoot: Dir = makeDir(baseDir);
 	routes
 		// Remove any entries that should be hidden
-		.filter((doc) => !doc.entry.data.sidebar.hidden)
+		.filter((doc) => (doc.entry.data.sidebar === undefined || !doc.entry.data.sidebar.hidden))
 		// Compute the path of each entry from the root of the collection ahead of time.
 		.map((doc) => [getRoutePathRelativeToCollectionRoot(doc, locale), doc] as const)
 		// Sort by depth, to build the tree depth first.
@@ -277,9 +277,9 @@ function treeify(routes: Route[], locale: string | undefined, baseDir: string):
 function linkFromRoute(route: Route, attrs?: LinkHTMLAttributes): SidebarLink {
 	return makeSidebarLink(
 		slugToPathname(route.slug),
-		route.entry.data.sidebar.label || route.entry.data.title,
-		route.entry.data.sidebar.badge,
-		{ ...attrs, ...route.entry.data.sidebar.attrs }
+		route.entry.data.sidebar?.label || route.entry.data.title,
+		route.entry.data.sidebar?.badge,
+		{ ...attrs, ...route.entry.data.sidebar?.attrs }
 	);
 }
 
@@ -291,7 +291,7 @@ function getOrder(routeOrDir: Route | Dir): number {
 	return isDir(routeOrDir)
 		? Math.min(...Object.values(routeOrDir).flatMap(getOrder))
 		: // If no order value is found, set it to the largest number possible.
-			(routeOrDir.entry.data.sidebar.order ?? Number.MAX_VALUE);
+        (routeOrDir.entry.data.sidebar?.order ?? Number.MAX_VALUE);;
 }
 
 /** Sort a directory’s entries by user-specified order or alphabetically if no order specified. */
diff --git a/node_modules/@astrojs/starlight/utils/routing/index.ts b/node_modules/@astrojs/starlight/utils/routing/index.ts
index 460d15a..a47d49c 100644
--- a/node_modules/@astrojs/starlight/utils/routing/index.ts
+++ b/node_modules/@astrojs/starlight/utils/routing/index.ts
@@ -42,7 +42,7 @@ export function normalizeCollectionEntry(entry: StarlightDocsCollectionEntry): S
 const docs: StarlightDocsEntry[] = (
 	(await getCollection('docs', ({ data }) => {
 		// In production, filter out drafts.
-		return import.meta.env.MODE !== 'production' || data.draft === false;
+		return import.meta.env.MODE !== 'production' || data.draft != true;
 	})) ?? []
 ).map(normalizeCollectionEntry);
 
